types/index.d.ts

```diff
@@ -284,6 +293,7 @@ declare interface Visualizer {
 declare interface Execution {
     request: any;
     response: any;
+    message: any;
```

Comment:
Isn't this interface for `pm.execution`? If so, messages shouldn't be part of it.